src/lib.rs

```diff
@@ -623,6 +623,33 @@ impl<T, C: cfg::Config> Slab<T, C> {
         })
     }
 
+    /// Return a mutable reference to the value associated with the given key.
+    ///
+    /// This call borrows self mutably (at compile-time) which guarantees that we
```

Comment:
Teensy nit: I might format `self` in monospace here:
```suggestion
    /// This call borrows `self` mutably (at compile-time) which guarantees that we
```